content/docs/design/visual_story/setting_up.md

```diff
@@ -40,12 +40,6 @@ http://localhost:8000/pets-completed.html
 Make sure the URL serves from `localhost` otherwise the AMP story might not load correctly, and you may encounter errors like `"source" "must start with "https://" or "//" or be relative and served from either https or from localhost.`
 {% endcall %}
 
-{% call callout('Important', type='caution') %}
-You may encounter a screen that states "<b>You must enable the amp-story experiment to view this content</b>". The amp-story component is currently [experimental](/docs/reference/experimental.html).  To view the story, you must click the **Enable** button to turn on the amp-story component for your domain, and then reload the page. Although not necessary for this tutorial, you might want to opt your browser into the AMP Dev Channel, which allows your browser to use a newer, experimental version of the AMP JS libraries.  You can learn more about the [AMP Dev Channel here](/docs/reference/experimental.html).
```

Comment:
So, this isn't true anymore?  That is, you don't need to enable the "amp-story" experiment for stories? @newmuis